app-config.yaml

```diff
@@ -47,3 +47,10 @@ catalog:
       target: ../../mock-catalog/templates/create-postgres-database.yaml
     - type: file
       target: ../../mock-catalog/templates/create-python-module.yaml
+
+slackCatalog:
+  token: ${SLACK_API_TOKEN_CATALOG}
+
+gitlabCatalog:
```

Comment:
added these mostly for convenience, and it doesn't hurt to have it added in the demo backstage